File: PYTHON PROGRAMMING/Test6.py
# 6. RETURNING MULTIPLE VALUES IN A FUNCTION
def multFunc(ip1, ip2):
    Add = ip1 + ip2
    Sub = ip1 - ip2
    Mult = ip1 * ip2
    Div = ip1 / ip2
    Mod = ip1 % ip2
    return Add, Sub, Mult, Div, Mod

add, sub, mult, div, mod = multFunc(250, 13)
print("Addition:", add)
print("Subtraction:", sub)
print("Multiplication:", mult)
print("Division:", div)
print("Modulus:", mod)

File: TYPESCRIPT PROGRAMMING/Test6.ts
// 6. RETURNING MULTIPLE VALUES IN A FUNCTION
const func = (ip1: number, ip2: number): {add: number; sub: number; mult: number; div: number; mod: number;} => {
    let ADD = ip1 + ip2;
    let SUB = ip1 - ip2;
    let MULT = ip1 * ip2;
    let DIV = ip1 / ip2;
    let MOD = ip1 % ip2;

    return {
        add: ADD,
        sub: SUB,
        mult: MULT,
        div: DIV,
        mod: MOD
    };
};
const res = func(125, 3);
console.log(res.add);
console.log(res.sub);
console.log(res.mult);
console.log(res.div);
console.log(res.mod);